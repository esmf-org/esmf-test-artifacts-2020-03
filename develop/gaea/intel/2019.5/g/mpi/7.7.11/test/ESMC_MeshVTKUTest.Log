build time -- 2021-12-18 00:10:24
20211218 002656.622 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211218 002656.622 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211218 002656.622 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211218 002656.622 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211218 002656.622 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211218 002656.622 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211218 002656.622 INFO             PET0 Running with ESMF Version   : 8.3.0 beta snapshot
20211218 002656.622 INFO             PET0 ESMF library build date/time: "Dec 18 2021" "00:08:25"
20211218 002656.622 INFO             PET0 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20211218 002656.622 INFO             PET0 ESMF_COMM                   : mpi
20211218 002656.623 INFO             PET0 ESMF_MOAB                   : enabled
20211218 002656.623 INFO             PET0 ESMF_LAPACK                 : enabled
20211218 002656.623 INFO             PET0 ESMF_NETCDF                 : enabled
20211218 002656.623 INFO             PET0 ESMF_PNETCDF                : disabled
20211218 002656.623 INFO             PET0 ESMF_PIO                    : enabled
20211218 002656.623 INFO             PET0 ESMF_YAMLCPP                : enabled
20211218 002656.623 INFO             PET0 Beginning Test, file ESMC_MeshVTKUTest.C, line 63
20211218 002656.623 INFO             PET0 NUMBER_OF_PROCESSORS 6
20211218 002656.623 INFO             PET0 PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
20211218 002656.625 INFO             PET0 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
20211218 002656.626 INFO             PET0 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
20211218 002656.635 INFO             PET0 PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
20211218 002656.635 INFO             PET0 PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
20211218 002656.635 INFO             PET0 PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
20211218 002656.652 INFO             PET0 PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
20211218 002656.652 INFO             PET0 PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
20211218 002656.652 INFO             PET0 PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
20211218 002656.652 INFO             PET0 PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
20211218 002656.671 INFO             PET0 PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
20211218 002656.671 INFO             PET0 PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
20211218 002656.671 INFO             PET0 PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
20211218 002656.671 INFO             PET0 Ending Test, file ESMC_MeshVTKUTest.C, line 301
20211218 002656.671 INFO             PET0 Finalizing ESMF
20211218 002656.636 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211218 002656.637 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211218 002656.637 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211218 002656.637 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211218 002656.637 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211218 002656.637 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211218 002656.637 INFO             PET1 Running with ESMF Version   : 8.3.0 beta snapshot
20211218 002656.637 INFO             PET1 ESMF library build date/time: "Dec 18 2021" "00:08:25"
20211218 002656.637 INFO             PET1 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20211218 002656.637 INFO             PET1 ESMF_COMM                   : mpi
20211218 002656.637 INFO             PET1 ESMF_MOAB                   : enabled
20211218 002656.638 INFO             PET1 ESMF_LAPACK                 : enabled
20211218 002656.638 INFO             PET1 ESMF_NETCDF                 : enabled
20211218 002656.638 INFO             PET1 ESMF_PNETCDF                : disabled
20211218 002656.638 INFO             PET1 ESMF_PIO                    : enabled
20211218 002656.638 INFO             PET1 ESMF_YAMLCPP                : enabled
20211218 002656.638 INFO             PET1 Beginning Test, file ESMC_MeshVTKUTest.C, line 63
20211218 002656.638 INFO             PET1 NUMBER_OF_PROCESSORS 6
20211218 002656.638 INFO             PET1 PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
20211218 002656.639 INFO             PET1 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
20211218 002656.640 INFO             PET1 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
20211218 002656.648 INFO             PET1 PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
20211218 002656.648 INFO             PET1 PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
20211218 002656.648 INFO             PET1 PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
20211218 002656.652 INFO             PET1 PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
20211218 002656.652 INFO             PET1 PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
20211218 002656.652 INFO             PET1 PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
20211218 002656.652 INFO             PET1 PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
20211218 002656.673 INFO             PET1 PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
20211218 002656.673 INFO             PET1 PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
20211218 002656.673 INFO             PET1 PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
20211218 002656.673 INFO             PET1 Ending Test, file ESMC_MeshVTKUTest.C, line 301
20211218 002656.673 INFO             PET1 Finalizing ESMF
20211218 002656.627 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211218 002656.627 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211218 002656.627 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211218 002656.627 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211218 002656.627 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211218 002656.628 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211218 002656.628 INFO             PET2 Running with ESMF Version   : 8.3.0 beta snapshot
20211218 002656.628 INFO             PET2 ESMF library build date/time: "Dec 18 2021" "00:08:25"
20211218 002656.628 INFO             PET2 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20211218 002656.628 INFO             PET2 ESMF_COMM                   : mpi
20211218 002656.629 INFO             PET2 ESMF_MOAB                   : enabled
20211218 002656.629 INFO             PET2 ESMF_LAPACK                 : enabled
20211218 002656.629 INFO             PET2 ESMF_NETCDF                 : enabled
20211218 002656.629 INFO             PET2 ESMF_PNETCDF                : disabled
20211218 002656.629 INFO             PET2 ESMF_PIO                    : enabled
20211218 002656.629 INFO             PET2 ESMF_YAMLCPP                : enabled
20211218 002656.629 INFO             PET2 Beginning Test, file ESMC_MeshVTKUTest.C, line 63
20211218 002656.629 INFO             PET2 NUMBER_OF_PROCESSORS 6
20211218 002656.629 INFO             PET2 PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
20211218 002656.630 INFO             PET2 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
20211218 002656.632 INFO             PET2 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
20211218 002656.640 INFO             PET2 PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
20211218 002656.640 INFO             PET2 PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
20211218 002656.640 INFO             PET2 PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
20211218 002656.652 INFO             PET2 PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
20211218 002656.652 INFO             PET2 PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
20211218 002656.652 INFO             PET2 PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
20211218 002656.652 INFO             PET2 PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
20211218 002656.666 INFO             PET2 PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
20211218 002656.667 INFO             PET2 PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
20211218 002656.667 INFO             PET2 PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
20211218 002656.667 INFO             PET2 Ending Test, file ESMC_MeshVTKUTest.C, line 301
20211218 002656.667 INFO             PET2 Finalizing ESMF
20211218 002656.638 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211218 002656.638 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211218 002656.638 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211218 002656.638 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211218 002656.638 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211218 002656.638 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211218 002656.638 INFO             PET3 Running with ESMF Version   : 8.3.0 beta snapshot
20211218 002656.638 INFO             PET3 ESMF library build date/time: "Dec 18 2021" "00:08:25"
20211218 002656.638 INFO             PET3 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20211218 002656.638 INFO             PET3 ESMF_COMM                   : mpi
20211218 002656.639 INFO             PET3 ESMF_MOAB                   : enabled
20211218 002656.639 INFO             PET3 ESMF_LAPACK                 : enabled
20211218 002656.639 INFO             PET3 ESMF_NETCDF                 : enabled
20211218 002656.639 INFO             PET3 ESMF_PNETCDF                : disabled
20211218 002656.639 INFO             PET3 ESMF_PIO                    : enabled
20211218 002656.639 INFO             PET3 ESMF_YAMLCPP                : enabled
20211218 002656.639 INFO             PET3 Beginning Test, file ESMC_MeshVTKUTest.C, line 63
20211218 002656.639 INFO             PET3 NUMBER_OF_PROCESSORS 6
20211218 002656.639 INFO             PET3 PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
20211218 002656.641 INFO             PET3 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
20211218 002656.642 INFO             PET3 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
20211218 002656.649 INFO             PET3 PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
20211218 002656.649 INFO             PET3 PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
20211218 002656.649 INFO             PET3 PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
20211218 002656.652 INFO             PET3 PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
20211218 002656.652 INFO             PET3 PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
20211218 002656.652 INFO             PET3 PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
20211218 002656.652 INFO             PET3 PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
20211218 002656.682 INFO             PET3 PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
20211218 002656.683 INFO             PET3 PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
20211218 002656.683 INFO             PET3 PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
20211218 002656.683 INFO             PET3 Ending Test, file ESMC_MeshVTKUTest.C, line 301
20211218 002656.683 INFO             PET3 Finalizing ESMF
20211218 002656.628 INFO             PET4 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211218 002656.628 INFO             PET4 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211218 002656.628 INFO             PET4 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211218 002656.628 INFO             PET4 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211218 002656.628 INFO             PET4 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211218 002656.628 INFO             PET4 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211218 002656.628 INFO             PET4 Running with ESMF Version   : 8.3.0 beta snapshot
20211218 002656.628 INFO             PET4 ESMF library build date/time: "Dec 18 2021" "00:08:25"
20211218 002656.628 INFO             PET4 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20211218 002656.628 INFO             PET4 ESMF_COMM                   : mpi
20211218 002656.629 INFO             PET4 ESMF_MOAB                   : enabled
20211218 002656.629 INFO             PET4 ESMF_LAPACK                 : enabled
20211218 002656.629 INFO             PET4 ESMF_NETCDF                 : enabled
20211218 002656.629 INFO             PET4 ESMF_PNETCDF                : disabled
20211218 002656.629 INFO             PET4 ESMF_PIO                    : enabled
20211218 002656.629 INFO             PET4 ESMF_YAMLCPP                : enabled
20211218 002656.629 INFO             PET4 Beginning Test, file ESMC_MeshVTKUTest.C, line 63
20211218 002656.629 INFO             PET4 NUMBER_OF_PROCESSORS 6
20211218 002656.629 INFO             PET4 PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
20211218 002656.631 INFO             PET4 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
20211218 002656.633 INFO             PET4 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
20211218 002656.642 INFO             PET4 PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
20211218 002656.643 INFO             PET4 PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
20211218 002656.643 INFO             PET4 PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
20211218 002656.652 INFO             PET4 PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
20211218 002656.652 INFO             PET4 PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
20211218 002656.652 INFO             PET4 PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
20211218 002656.652 INFO             PET4 PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
20211218 002656.659 INFO             PET4 PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
20211218 002656.659 INFO             PET4 PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
20211218 002656.659 INFO             PET4 PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
20211218 002656.659 INFO             PET4 Ending Test, file ESMC_MeshVTKUTest.C, line 301
20211218 002656.659 INFO             PET4 Finalizing ESMF
20211218 002656.622 INFO             PET5 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211218 002656.622 INFO             PET5 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211218 002656.622 INFO             PET5 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211218 002656.622 INFO             PET5 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211218 002656.622 INFO             PET5 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211218 002656.622 INFO             PET5 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211218 002656.622 INFO             PET5 Running with ESMF Version   : 8.3.0 beta snapshot
20211218 002656.622 INFO             PET5 ESMF library build date/time: "Dec 18 2021" "00:08:25"
20211218 002656.622 INFO             PET5 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20211218 002656.622 INFO             PET5 ESMF_COMM                   : mpi
20211218 002656.623 INFO             PET5 ESMF_MOAB                   : enabled
20211218 002656.623 INFO             PET5 ESMF_LAPACK                 : enabled
20211218 002656.623 INFO             PET5 ESMF_NETCDF                 : enabled
20211218 002656.623 INFO             PET5 ESMF_PNETCDF                : disabled
20211218 002656.623 INFO             PET5 ESMF_PIO                    : enabled
20211218 002656.623 INFO             PET5 ESMF_YAMLCPP                : enabled
20211218 002656.623 INFO             PET5 Beginning Test, file ESMC_MeshVTKUTest.C, line 63
20211218 002656.623 INFO             PET5 NUMBER_OF_PROCESSORS 6
20211218 002656.623 INFO             PET5 PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
20211218 002656.625 INFO             PET5 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
20211218 002656.626 INFO             PET5 PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
20211218 002656.635 INFO             PET5 PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
20211218 002656.636 INFO             PET5 PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
20211218 002656.636 INFO             PET5 PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
20211218 002656.652 INFO             PET5 PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
20211218 002656.652 INFO             PET5 PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
20211218 002656.652 INFO             PET5 PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
20211218 002656.652 INFO             PET5 PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
20211218 002656.664 INFO             PET5 PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
20211218 002656.664 INFO             PET5 PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
20211218 002656.664 INFO             PET5 PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
20211218 002656.664 INFO             PET5 Ending Test, file ESMC_MeshVTKUTest.C, line 301
20211218 002656.664 INFO             PET5 Finalizing ESMF
