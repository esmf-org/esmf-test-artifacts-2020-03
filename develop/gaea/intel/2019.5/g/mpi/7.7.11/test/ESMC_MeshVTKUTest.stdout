build time -- 2022-01-23 00:10:35
Beginning Test, file ESMC_MeshVTKUTest.C, line 63
NUMBER_OF_PROCESSORS 6
PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
Beginning Test, file ESMC_MeshVTKUTest.C, line 63
NUMBER_OF_PROCESSORS 6
PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
Beginning Test, file ESMC_MeshVTKUTest.C, line 63
NUMBER_OF_PROCESSORS 6
PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
Beginning Test, file ESMC_MeshVTKUTest.C, line 63
NUMBER_OF_PROCESSORS 6
Beginning Test, file ESMC_MeshVTKUTest.C, line 63
NUMBER_OF_PROCESSORS 6
PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
Beginning Test, file ESMC_MeshVTKUTest.C, line 63
NUMBER_OF_PROCESSORS 6
PASS MeshCreate, ESMC_MeshVTKUTest.C, line 80
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 89
PASS MeshVTKHeader, ESMC_MeshVTKUTest.C, line 110
PASS MeshVTKBody, ESMC_MeshVTKUTest.C, line 129
PASS MeshAddNodes, ESMC_MeshVTKUTest.C, line 158
PASS MeshGetLocalNodeCount, ESMC_MeshVTKUTest.C, line 178
PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
PASS MeshAddElements, ESMC_MeshVTKUTest.C, line 187
PASS MeshGetLocalElementCount, ESMC_MeshVTKUTest.C, line 207
PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
PASS MeshGetOwnedElementCount, ESMC_MeshVTKUTest.C, line 228
PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
PASS MeshGetOwnedNodeCount, ESMC_MeshVTKUTest.C, line 249
PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
Ending Test, file ESMC_MeshVTKUTest.C, line 301
PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
Ending Test, file ESMC_MeshVTKUTest.C, line 301
PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
Ending Test, file ESMC_MeshVTKUTest.C, line 301
PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
Ending Test, file ESMC_MeshVTKUTest.C, line 301
PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
Ending Test, file ESMC_MeshVTKUTest.C, line 301
PASS MeshWrite, ESMC_MeshVTKUTest.C, line 269
PASS MeshFreeMemory, ESMC_MeshVTKUTest.C, line 279
PASS MeshDestroy, ESMC_MeshVTKUTest.C, line 288
Ending Test, file ESMC_MeshVTKUTest.C, line 301
 PET 4 Test Elapsed Time  218.217000 msec.
 PET 2 Test Elapsed Time  212.224000 msec.
 PET 3 Test Elapsed Time  201.257000 msec.
 PET 1 Test Elapsed Time  183.687000 msec.
 PET 5 Test Elapsed Time  213.363000 msec.
 PET 0 Test Elapsed Time  201.349000 msec.
